dts/bindings/can/xlnx,can.yaml

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2022, Weidmueller Interface GmbH & Co. KG
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: |
+  Xilinx Processor System CAN controller
+
+  This CAN controller is contained in both the Xilinx Zynq-7000 and
+  ZynqMP (UltraScale) SoCs.
+
+compatible: "xlnx,can"
```

Comment:
Please use the same compatible as used in the upstream Linux devicetree bindings ("xlnx,zynq-can-1.0"). See https://elixir.bootlin.com/linux/latest/source/Documentation/devicetree/bindings/net/can/xilinx_can.txt